### PR TITLE
bundle css and license file into archive

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: css/core.css [hunk 1]
```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
+/*! Bassplate | MIT License | http://github.com/basscss/bassplate */
```
Comment (camelburrito) [inline css/core.css]: this should not go before normalize

File: tasks/bundle.js [hunk 2]
```diff
@@ -66,3 +86,52 @@ function bundle() {
 }
 
 gulp.task('bundle', bundle);
+
+const licenses = `
+# The MIT License (MIT)
+
```
Comment (camelburrito) [inline tasks/bundle.js]: Should we be adding a heading to associate license to the respective projects?

File: css/core.css (hunk 1 above)
Comment (erwinmombay) [inline css/core.css]: we need both of them at the top

File: tasks/bundle.js (hunk 2 above)
Comment (erwinmombay) [inline tasks/bundle.js]: not sure how it works honestly but it makes sense to do that

Comment (camelburrito) [inline tasks/bundle.js]: Could you just do 

Basscss | https://github.com/basscss/basscss/blob/master/LICENSE.md
---------------------------------
ACTUAL Licence Text

Normalize | https://github.com/necolas/normalize.css/blob/master/LICENSE.md
---------------------------------
Actual license text

**Pull request merged**